pipettes/firmware/stm32g4xx_it.c

```diff
@@ -135,7 +136,22 @@ void FDCAN1_IT0_IRQHandler(void)
     /* USER CODE END FDCAN1_IT0_IRQn 1 */
 }
 
+/**
+  * @brief This function handles EXTI line[15:10] interrupts.
+  */
+void EXTI15_10_IRQHandler(void)
```

Comment:
what's this for?

Reply:
We don't need it. It was something added by the ST Cube IDE.